CMakeLists.txt

```diff
@@ -184,11 +186,24 @@ set(enable_camera "false")
 set(enable_wind "false")
 set(rotors_description_dir "${CMAKE_CURRENT_SOURCE_DIR}/models/rotors_description")
 set(scripts_dir "${CMAKE_CURRENT_SOURCE_DIR}/scripts")
+# set the vision estimation to be sent if set by the CMake option SEND_VISION_ESTIMATION_DATA
+set(send_vision_estimation "false")
+if (SEND_VISION_ESTIMATION_DATA)
+  set(send_vision_estimation "true")
+endif()
+
+# if SEND_ODOMETRY_DATA option is set, then full odometry data is sent instead of
+# only the visual pose estimate
+set(send_odometry "false")
+if (SEND_ODOMETRY_DATA)
+  set(send_odometry "true")
+  set(send_vision_estimation "false")
```

Comment:
So VISION_ESTIMATION is purposefully **not** sent when ODOMETRY is sent?

Reply:
correct

Reply:
They are mutually exclusive in this situation.